tests/jsonpickle_test.py

```diff
@@ -257,6 +257,12 @@ def test_list(self):
         self.assertEqual(listD, self.pickler.flatten(listD))
         self.assertEqual(listD, self.unpickler.restore(listD))
 
+    def test_NoneType(self):
```

Comment:
Can you name this ``test_nonetype``? Mixing snake_case and PascalCase, while it's accurate to both the stdlib name and jsonpickle's general formatting, is a bit jarring to read. Apart from that, thanks so much for writing a test! Most first-time contributors don't do that, I forgot when it was my first PR!

Reply:
That's reasonable. Addressed this in an update to the PR. Thanks for a quick review!